vernac/loadpath.mli

```diff
@@ -88,6 +88,10 @@ type vo_path =
       true for -R, false for -Q in command line *)
   ; recursive : bool
   (** [recursive] will determine whether we explore sub-directories  *)
+  ; automatic : bool
```

Comment:
`automatic` seems quite non-standard, also IIANM `ROCQPATH` will set this to true, which is not so "automatic".

In build systems this is called `installed`, I dunno, `global` could be another option. It seems to me that `installed` better fits the two use cases.